apps/java-maven/mvnw

```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    https://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
ewat?

Reply:
/shrug, I just updated to the latest version of the maven wrapper.

Ref: https://github.com/takari/maven-wrapper/blob/master/mvnw